tf_encrypted/convert/convert.py

```diff
@@ -94,7 +94,17 @@ def convert(
           self.outputs[output] = x
           continue
 
-        self.outputs[output] = register[node.op](self, node, inputs)
+        out = register[node.op](self, node, inputs)
+
+        # if the operation returns a list with several ouputs,
+        # identify the outputs node name
+        if isinstance(out, list):
```

Comment:
can we switch this to `isinstance(out, (list, tuple))`? feels a bit more robust

Reply:
definitely, good point.